packages/storybook/stories/va-alert.stories.jsx

```diff
@@ -60,8 +64,26 @@ const Template = ({
   closeable,
   'full-width': fullWidth,
   headline,
-  onclose,
+  vaClose,
```

Comment:
I think it might be clearer if this was `onVaClose` just to indicate that it is a handler.

Reply:
Updated it. 👍